packages/api/src/agent.ts

```diff
@@ -61,15 +61,6 @@ const THREAD_VIEW_PREF_DEFAULTS = {
   prioritizeFollowedUsers: true,
 }
 
-declare global {
-  interface Array<T> {
-    findLast(
-      predicate: (value: T, index: number, obj: T[]) => unknown,
-      thisArg?: any,
-    ): T
-  }
-}
-
```

Comment:
This is not the proper type def for this method. The correct signature is provided by ts's 2023 lib.

Since we are relying on this method, I updated the tsconfig to use 2023 lib instead of 2022.